tests/js/utils.js

```diff
@@ -627,6 +627,7 @@ export const waitForDefaultTimeouts = () => {
 
 /**
  * Creates a delay in the execution of subsequent code for a specified duration in milliseconds.
+ * Developers should consider using waitForRegistry() instead if state changes occur.
```

Comment:
Just a small formatting tweak here. 😅 

```suggestion
 *
 * Developers should consider using `waitForRegistry()`, instead of this helper,
 * if state changes occur.
```